.github/actions/staging-deploy/action.yml

```diff
@@ -0,0 +1,157 @@
+### !!!
+# This file is automatically generated using Terraform.
+# Do not update it manually. If changes need to be made,
+# please request help from a maintainer to generate the
+# updated version.
+### !!!
+
+name: openverse/staging-nuxt-nuxt-deploy
```

Comment:
This nuxt-nuxt is a consequence of the infra's naming structure and can be changed in the future.